axiom/cmd_appinfo_transmit.c

```diff
@@ -178,6 +178,12 @@ nr_flatbuffer_t* nr_appinfo_create_query(const char* agent_run_id,
                                     info->span_queue_size, 0);
   nr_flatbuffers_object_prepend_u64(fb, APP_SPAN_EVENTS_MAX_SAMPLES_STORED,
                                     info->span_events_max_samples_stored, 0);
+  nr_flatbuffers_object_prepend_u64(fb, APP_LOG_EVENTS_MAX_SAMPLES_STORED,
+                                    info->log_events_max_samples_stored, 0);
+  // MSF - POSSIBLE REMOVE FROM RELEASE
```

Comment:
I would remove this comment for cleanliness
```suggestion
```

Reply:
Responding to https://github.com/newrelic/newrelic-php-agent/pull/497#issuecomment-1204392306

I think it may be better to remove these now and add debug statements as needed during dev/testing, otherwise we may forget these are here until after the release 😅

Reply:
I've cleaned up these debug messages and have left the ones which I feel are relevant for troubleshooting.  
The changes are in commit [a198099](https://github.com/newrelic/newrelic-php-agent/pull/497/commits/a198099168e2b57511fb842827feba16138366d4).